### PR TITLE
chromium: add page

### PR DESCRIPTION
**Pull request by BinaryBrain**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/chromium.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# chromium
+
+> Open-source web browser from Google.
+
+- Open a file:
+
+`chromium {{file.html}}`
```
Comment (sbrl) [inline pages/common/chromium.md]: Could `file.html` be `path/to/file.html` please?

File: pages/common/chromium.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`chromium --new-window {{example.com}}`
+
+- Open in app mode (without toolbars, URL bar, buttons, etc.):
+
+`chromium --app='{{https://example.com}}'`
+
+- Use a proxy server:
+
+`chromium --proxy-server="{{socks5://foobar:66}}" {{example.com}}`
```
Comment (agnivade) [inline pages/common/chromium.md]: please change foobar to hostname.

**Comment by BinaryBrain** [comment]
@agnivade Because the protocol is optional everywhere except with the `--app` option apparently.
So if we add the protocol everywhere as you suggest, the use may not know it is optional.

**Comment by agnivade** [comment]
Thanks for clarifying !

**Pull request merged**